bisection.py

```diff
@@ -0,0 +1,373 @@
+"""bisection.py
+Runs bisection to determine PRs that cause performance regression.
+Performance regression is defined by TorchBench score drop greater than the threshold.
+By default, the torchvision, torchaudio, and torchtext package version will be fixed to the latest git version.
+
+Usage:
+  python bisection.py --pytorch-src <PYTORCH_SRC_DIR> \
```

Comment:
does this script assume that PYTORCH_SRC_DIR is a git repo, and in a 'clean' state, and it's OK to change branches on it?

Reply:
Yes it does. Maybe I should check the repo state before modifying it.